src/pollution_model/run_medusa_2.py

```diff
@@ -19,6 +20,7 @@
 from src import geoserver
 from src.config import EnvVariable
 from src.digitaltwin import setup_environment
+from src.digitaltwin import tables
 from src.digitaltwin.tables import create_table, check_table_exists
```

Comment:
It looks like this import is already covered by `from src.digitaltwin.tables import create_table, check_table_exists`, so use that format instead of introducing more imports.